### ISSUE TITLE
Tablist: ARIA tab design states that ARROW key is for tab traversal while TAB key should be reserved for shifting focus onto the tab content

### ISSUE THREAD

**Issue by ceszilla** [infor-design/enterprise#4745]
**Describe the bug**
[VPAT] 
ARIA tab design specification states that ARROW keys are for navigating focus thru tabs while TAB key should be reserved for shifting focus onto the tab content, and should not behave like the ARROW key in this scenario and perform tab traversal as well. This is based on the expectation of screen readers.

According to the report:
The aria Tabs design pattern is intended to allow users to navigate tabs using arrow keys, and use the Tab key to navigate into the tab panel. Screen reader users will not expect the Tab key to move their focus to the next navigation element/tab. 

This can be done by specifying tabindex=-1 
Please see: 
https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_roving_tabindex

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/tabs/example-index.html
2. Use ARROW key to navigate thru tabs -- correct
3. Use TAB key to shift focus to the content of the currently selected tab -- incorrect, currently TAB also functions like the ARROW key

**Expected behavior**
In tablist, the TAB key should shift focus onto the content of selected tab and not perform tab focus traversal like the ARROW key

**Version**
- ids-enterprise: 4.37

**Screenshots**

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable): N/A
 - OS Version: Windows 10 Ent
 - Browser Name: Chrome
 - Browser Version: 87.0.4280.88 (Official Build) (64-bit)

**Additional context**
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

**Comment by tmcconechy** [comment]
Seems like a legit bug but just wondering if its because there is no content in these examples.

**Comment by ceszilla** [comment]
hi Tim. the example pages in the original ticket are not empty. ill add screenshots in the JT and tag you. thanks

**Comment by ceszilla** [comment]
one of the identified high priority VPAT items Tim. thanks

**Comment by janahintal** [comment]
QA Passed. 
Verified in 
https://4370-beta0-enterprise.demo.design.infor.com/components/tabs-vertical/example-index.html
https://4370-beta0-enterprise.demo.design.infor.com/components/tabs-module/example-app-menu-button-audible#order-123456
https://4370-beta0-enterprise.demo.design.infor.com/components/tabs/example-index
